inscriptions/ISic003869.xml

```diff
@@ -77,26 +74,30 @@
                         <objectDesc>
                             <supportDesc>
                                 <support>
-                                    <dimensions><!--Default values and requires editing-->
-                                        <height unit="cm"/>
-                                        <width unit="cm"/>
-                                        <depth unit="cm"/>
+                                    <p>Smooth stele slightly tapered at the top, with a rough base, more expanded on the front and on the left side. Missing the upper left corner, with extensive chipping in the upper part of the front face.</p>
+                                    <p>The stele has been reused as a funerary monument twice and is an opistograph. The oldest inscription is presented here. For the more recent one, see I.Lipara 403 (<ref target="http://sicily.classics.ox.ac.uk/inscription/ISic004007">ISic004007</ref>).</p>
```

Comment:
now using the model of `<p>... same support as <ref sameAs="ISic004007.xml">ISic004007 (I.Lipara 403)</ref></p>`